documentation/modules/overview/con-security-configuration-authorization.adoc

```diff
@@ -16,9 +16,11 @@ Supported authorization mechanisms:
 * Simple authorization
 * OAuth 2.0 authorization (if you are using OAuth 2.0 token-based authentication)
 * Open Policy Agent (OPA) authorization
+* Custom authorization
 
 Simple authorization uses `AclAuthorizer`, the default Kafka authorization plugin.
 `AclAuthorizer` uses Access Control Lists (ACLs) to define which users have access to which resources.
+For custom authorization, you configure your own `Authorizer` plugin to define ACLs. 
```

Comment:
I think _define ACLs_ sounds a bit weird. The authorizer rather enforces the ACLs.

Reply:
Updated to:

> For custom authorization, you configure your own `Authorizer` plugin to enforce ACL rules.